dotnet/samples/Hello/HelloAgent/Program.cs

```diff
@@ -4,14 +4,23 @@
 using Microsoft.AutoGen.Agents;
 using Microsoft.AutoGen.Contracts;
 using Microsoft.AutoGen.Core;
+using Microsoft.AutoGen.Core.Grpc;
 using Samples;
-
-// Set up app builder for in-process runtime, allow message delivery to self, and add the Hello agent
-AgentsAppBuilder appBuilder = new AgentsAppBuilder()
-    .UseInProcessRuntime(deliverToSelf: true)
-    .AddAgent<HelloAgent>("HelloAgent");
+var appBuilder = new AgentsAppBuilder(); // Create app builder
+// if we are using distributed, we need the AGENT_HOST var defined and then we will use the grpc runtime
+if (Environment.GetEnvironmentVariable("AGENT_HOST") != null)
```

Comment:
[Minor] A thing you can do here is 

```suggestion
if (Environment.GetEnvironmentVariable("AGENT_HOST") is string agentHost)
```

And then you will not need to reach out to the environment again for it.